app/code/core/Mage/Core/Model/Resource/Abstract.php

```diff
@@ -197,7 +197,7 @@ protected function _unserializeField(Varien_Object $object, $field, $defaultValu
         if (empty($value)) {
             $object->setData($field, $defaultValue);
         } elseif (!is_array($value) && !is_object($value)) {
-            $object->setData($field, unserialize($value));
+            $object->setData($field, unserialize($value, ['allowed_classes' => ['Varien_Object']]));
```

Comment:
yesterday I merged openmage into our codebase and this change broke several legitimate objects saved in the database resulting in an error `The script tried to execute a method or access a property of an incomplete object.`

this needs some changes for backward compatibility instead of forcing it to `['allowed_classes' => ['Varien_Object']]`

Reply:
@arthurpro can you please create an issue about it? Please include details about your use cases, how this can be reproduced. Thanks